src/org/zaproxy/zap/extension/bugTracker/RaiseManualIssueDialog.java

```diff
@@ -0,0 +1,105 @@
+/*
+ *
+ * Paros and its related class files.
+ * 
+ * Paros is an HTTP/HTTPS proxy for assessing web application security.
+ * Copyright (C) 2003-2004 Chinotec Technologies Company
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the Clarified Artistic License
+ * as published by the Free Software Foundation.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * Clarified Artistic License for more details.
+ * 
+ * You should have received a copy of the Clarified Artistic License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+ */
+package org.zaproxy.zap.extension.bugTracker;
+
+import java.awt.Frame;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.zaproxy.zap.utils.ZapTextField;
+ 
+import org.zaproxy.zap.view.StandardFieldsDialog;
+import java.awt.Dimension;
+
+public class RaiseManualIssueDialog extends StandardFieldsDialog {
```

Comment:
Yes. But I thought for now just for testing purposes I could keep it for a while. But okay I'll make a a different branch for that. :)